src/Autocomplete/assets/src/controller.ts

```diff
@@ -134,14 +134,15 @@ export default class extends Controller {
             // VERY IMPORTANT: use 'function (query, callback) { ... }' instead of the
             // '(query, callback) => { ... }' syntax because, otherwise,
             // the 'this.XXX' calls inside this method fail
-            load: function (query: string, callback: (results?: any) => void) {
+            load: function (query: string, callback: (options?: any, optgroups?: any) => void) {
                 const url = this.getUrl(query);
                 fetch(url)
                     .then(response => response.json())
                     // important: next_url must be set before invoking callback()
-                    .then(json => { this.setNextUrl(query, json.next_page); callback(json.results) })
+                    .then(json => { this.setNextUrl(query, json.next_page); callback(json.results.options || json.results, json.results.optgroups || []) })
```

Comment:
Instead of `json.results.options || json.results` can we just now use `json.results.options`? Or do we need to look for `json.results` still for some situation?

Also, can you add an entry to the CHANGELOG about the Ajax return format changing?